static/js/components/ChannelTracker.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import React from "react"
+import ReactGA from "react-ga"
+import type { Location } from "react-router"
+import type { Channel } from "../flow/discussionTypes"
+
+type ChannelTrackerProps = {
+  channel: Channel,
+  location: Location
+}
+
+export class ChannelTracker extends React.Component<ChannelTrackerProps> {
```

Comment:
I think this would be better done as an HOC so it doesn't have to be included in the page's render().